python-sdk/nuscenes/eval/detection/tests/test_utils.py

```diff
@@ -128,6 +128,19 @@ def rad(x):
         period = 360
         self.assertAlmostEqual(rad(180), abs(angle_diff(rad(a), rad(b), rad(period))))
 
+    def test_iou_complement_no_overlap(self):
+        # Two boxes specified, no overlap
+        sa = DetectionBox(translation=(1.0, 0.0, 1.0), size=(2,1,1))
+        sr = DetectionBox(translation=(3.5, 0.0, 1.0), size=(3,1,2))
+        self.assertAlmostEqual(iou_complement(sa, sr), 1.0)
+
+    def test_iou_complement_overlap(self):
```

Comment:
Can we follow the formatting of the other tests? E.g. have a single `def test_iou_complement(self)` and then have blocks of code for each sub-test